docroot/profiles/custom/cgov_site/modules/custom/cgov_video/config/install/core.entity_view_display.media.cgov_video.default.yml

```diff
@@ -160,6 +161,17 @@ content:
     third_party_settings: {  }
     weight: 25
     region: content
+  field_override_thumbnail_video:
```

Comment:
Is default being used somewhere specific? If not, then you should just hide this field.

Reply:
@navsunka I updated the requirements on the ticket so it broke out "Video Display: Full Content", "Video Display: Large", "Video Display: Medium", "Video Display: Small" to account for that part of Bryan's comment

Reply:
@navsunka this comment was resolved, but there was no comment or code change. Did you mark it resolved by accident?

Reply:
Hidden the override_thumbnail_video field for default.